replicaset.go

```diff
@@ -141,3 +135,40 @@ func (rs *Replicaset) ReplicaCall(
 		}, nil
 	}
 }
+
+type Future struct {
```

Comment:
now future simply actually repeats the behavior of Future inside go-tarantool, but it is not used in any way, I think it’s not worth adding this yet

Reply:
Done